nuxt.config.js

```diff
@@ -57,9 +57,7 @@ export default {
    ** Axios module configuration
    ** See https://axios.nuxtjs.org/options
    */
-  axios: {
-    baseURL: process.env.BASE_URL || 'http://localhost:3000/api',
```

Comment:
when will be axios configured?

Reply:
@liudavid0107 ☝️

Reply:
From Nuxt v2.13, they use runtime config.

Reply:
so, axios will be configured automatically from runtime config?

Reply:
Yes, I think so.